Parse/PFAnalytics.m

```diff
@@ -25,9 +25,11 @@ @implementation PFAnalytics
 ///--------------------------------------
 
 + (BFTask *)trackAppOpenedWithLaunchOptions:(NSDictionary *)launchOptions {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_WATCH
+    NSDictionary *userInfo = nil;
+#elif TARGET_OS_IOS
     NSDictionary *userInfo = launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
-#else
+#elif TARGET_OS_MAC
```

Comment:
I wish I did, but I can't think of anything off of the top of my head. Just stick with `PF_TARGET_OS_OSX`, and we can update that macro if we have any brilliant ideas on how to make it better later.